src/main/java/uk/gov/justice/digital/delius/utils/DateConverter.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.delius.utils;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+
+public class DateConverter {
+
+    private static final String LOCAL_TIMEZONE = "Europe/London";
```

Comment:
Maybe add a comment here to indicate why the timezone is hard-coded?

Reply:
Will add:

    // nDelius stores all dates/times relative to Europe/London (without any timezone/offset information). Consumers
    // to community-api (e.g. interventions) can provide their dates/times with offsets from GMT (OffsetDateTime).
    // These methods carry out the necessary conversion to Europe/London times as required by delius-api.
    // The timezone used in the conversion is therefore set and must not be changed.

Reply:
@mahalrmoj I'm still trying to get me head around this:

So without this change if a consumer for instance called the endpoint with 10am 1/1/2020 (during GMT) and then sent 10 am 1/6/202 (during BST) was Delius then recording the second time as 9am? Is this what was being observed?

Reply:
Yes that's right and specifically
1) Before BST and passes
```
        assertThat(OffsetDateTime.of(2021, 1, 1, 10, 0, 0, 0, ZoneOffset.UTC).toLocalDateTime())
            .isEqualTo(LocalDateTime.of(2021, 1, 1, 10, 0, 0, 0));
```
2) During BST 
```
        assertThat(OffsetDateTime.of(2021, 6, 1, 10, 0, 0, 0, ZoneOffset.UTC).toLocalDateTime())
            .isEqualTo(LocalDateTime.of(2021, 6, 1, 11, 0, 0, 0));
Expecting:
 <2021-06-01T10:00 (java.time.LocalDateTime)>
to be equal to:
 <2021-06-01T11:00 (java.time.LocalDateTime)>
```